roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -145,6 +145,19 @@ reserved_host_memory_mb=0
 {% elif inventory_hostname in groups['controller'] -%}
 reserved_host_memory_mb=4096
 {% endif %}
+
+{% if ceph.enabled and nova.backend_ceph -%}
+images_type = rbd
+images_rbd_pool = {{ ceph.vm_pool_name }}
+images_rbd_ceph_conf = {{ ceph.cinder.rbd_ceph_conf }}
+rbd_user = {{ ceph.cinder.rbd_user }}
+rbd_secret_uuid = {{ cinder_uuid_file.content | b64decode }}
```

Comment:
But this assumes that ceph will be used for cinder, which isn't necessarily the case. I'd prefer that these secrets be separated.